ntbs-service/deployments/training.yml

```diff
@@ -58,6 +58,9 @@ spec:
                 key: migrationDb
           - name: AzureAdOptions__Enabled
             value: "true"
+          # Read only users in other environments should still be able to do training
+          - name: AzureAdOptions__ReadOnlyUserGroup
+            value: "App.Auth.NIS.NTBS.Read_Only_DISABLED"
```

Comment:
The way that you've written this makes it seem like this is a hack, but it's actually just the feature behaving as intended. I think we should get rid of the comment, and rename the group to `App.Auth.NIS.NTBS.Read_Only_Training`.

Reply:
Yeah that makes sense, done!